flow360/cloud/flow360_requests.py

```diff
@@ -153,3 +164,67 @@ class NewReportRequest(Flow360RequestsV2):
     resources: List[_Resource]
     config_json: str
     solver_version: str
+
+
+class DraftPostRequest(Flow360RequestsV2):
+    """Data model for draft post request"""
+
+    name: Optional[str] = pd.Field(None)
+    project_id: IDStringType = pd.Field()
+    source_item_id: IDStringType = pd.Field()
+    source_item_type: Literal[
+        "Project", "Folder", "Geometry", "SurfaceMesh", "VolumeMesh", "Case", "Draft"
+    ] = pd.Field()
+    solver_version: str = pd.Field()
+    fork_case: bool = pd.Field()
+
+    @pd_v2.field_validator("name", mode="after")
+    @classmethod
+    def _generate_default_name(cls, values):
+        if values is None:
+            values = "Draft " + datetime.now().strftime("%m-%d %H:%M:%S")
+        return values
+
+
+class ForceCreationConfig(Flow360RequestsV2):
+    """Data model for force creation configuration"""
+
+    start_from: Literal["SurfaceMesh", "VolumeMesh", "Case"] = pd.Field()
+
+
+class DraftRunRequest(Flow360RequestsV2):
+    """Data model for draft run request"""
+
+    up_to: Literal["SurfaceMesh", "VolumeMesh", "Case"] = pd.Field()
+    use_in_house: bool = pd.Field()
+    force_creation_config: Optional[ForceCreationConfig] = pd.Field(
+        None,
+    )
+    source_item_type: Literal["Geometry", "SurfaceMesh", "VolumeMesh", "Case"] = pd.Field(
+        exclude=True
+    )
+
+    @pd_v2.model_validator(mode="after")
+    def _validate_force_creation_config(self):
+        # pylint: disable=no-member
+
+        order = {"Geometry": 0, "SurfaceMesh": 1, "VolumeMesh": 2, "Case": 3}
+        source_order = order[self.source_item_type]
+        up_to_order = order[self.up_to]
+
+        if self.force_creation_config is not None:
+            force_start_order = order[self.force_creation_config.start_from]
+            if force_start_order <= source_order < 3 or (
```

Comment:
Prefer using "Case" etc here instead of 3.

```suggestion
            if force_start_order <= source_order and self.source_item_type!="Case" or (
```

Reply:
OK, in this if check, replace all usage of 3 with "Case"